env_config.rb

```diff
@@ -30,9 +30,13 @@
     mandatory :VAULT_AWS_REGION, :string
     mandatory :TASK_ROLE, :string
     mandatory :WCA_REGISTRATIONS_URL, :string
-    mandatory :WCA_REGISTRATIONS_POLL_URL, :string
```

Comment:
just confirming this should be removed / that it won't have any effects on v2 in production?

Reply:
Handled in line 36 and at the end of the file